signer/src/config/default.toml

```diff
@@ -145,6 +145,31 @@ bootstrap_signatures_required = 2
 # TODO(715): Add default delay (15 seconds? see #701)
 bitcoin_processing_delay = 0
 
+# How many bitcoin blocks back from the chain tip the signer will
+# look for requests.
+# Optional.
+# Warning: this parameter may significantly affect how well the sBTC signer protocol operates
+# change it carefully
```

Comment:
You should use the same style of formatting as the other fields in this file. Also you should detail how to set them from the environment and which values are invalid.

Reply:
This basically good, but I think that we should note that any value less than 1 is invalid.